cmd/repo.go

```diff
@@ -302,7 +302,7 @@ func downloadIndex(url string) (*RepoIndex, error) {
 }
 
 func (index *RepoIndex) listProjects(repoName string, config *RootCommandConfig) (string, error) {
```

Comment:
While you are changing this code, can you come up with better names for the two `listProjects` methods?

Reply:
After looking again at these function, I believe this needs more work than just renaming. They should just be extract into a common function, since they share a lot of code.

Reply:
Fair enough. Please create a separate issue to fix this :)

Reply:
Just seen this! Yes, I'll create an issue to maybe refactor `repo.go`?